lib/order_repository.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'order'
+
+class OrderRepository
+  def all 
```

Comment:
The code is well-structured in the method. Some comments can be added to describe the method and the actions within.